models/command.js

```diff
@@ -0,0 +1,129 @@
+'use strict';
+
+const Joi = require('joi');
+const mutate = require('../lib/mutate');
+const Command = require('../config/command');
+
+const MODEL = {
+    id: Joi
+        .number().integer().positive()
+        .description('Identifier of this command')
+        .example(123345),
+
+    labels: Joi
+        .array()
+        .items(Joi.string())
+        .description('Labels for command')
+        .example(['stable', 'latest', 'beta']),
+
+    namespace: Command.namespace,
+    command: Command.command,
+    version: Command.version,
+    description: Command.description,
+    maintainer: Command.maintainer,
+    format: Command.format,
+    habitat: Command.habitat,
+    docker: Command.docker,
+    binary: Command.binary
+};
+
+module.exports = {
+    /**
+     * All the available properties of Command
+     *
+     * @property base
+     * @type {Joi}
+     */
+    base: Joi.object(MODEL).label('Command'),
+
+    /**
+     * Properties for command that will come back during a GET request
+     *
+     * @property get
+     * @type {Joi}
+     */
+    get: Joi.object(mutate(MODEL, [
+        'id',
+        'labels',
+        'namespace',
+        'command',
+        'version',
+        'description',
+        'maintainer',
+        'format'
+    ], [
+        'habitat',
+        'docker',
+        'binary'
+    ])).label('Get Command'),
+
+    /**
+     * Properties for command that will be passed during a CREATE request
+     *
+     * @property create
+     * @type {Joi}
+     */
+    create: Joi.object(mutate(MODEL, [
+        'namespace',
+        'command',
+        'version',
+        'description',
+        'maintainer',
+        'format'
+    ], [
+        'habitat',
+        'docker',
+        'binary',
+        'labels'
+    ])).label('Create Command'),
+
+    /**
+     * Properties for command that will be passed during a UPDATE request
+     *
+     * @property update
+     * @type {Joi}
+     */
+    update: Joi.object(mutate(MODEL, [], ['labels']))
```

Comment:
We probably don't need this if we remove `labels`

Reply:
Yes, I'll remove it.